app/components/skill-list-item.js

```diff
@@ -12,15 +13,13 @@ export default Component.extend({
   tagName: 'li',
 
   matched: notEmpty('userSkill'),
-  userSkills: service(),
+  userSkillsList: service(),
```

Comment:
As you said earlier, I think we'll loop back to this once we have to expand out to other cases.